charts/alfresco-common/templates/_helpers-nginx.tpl

```diff
@@ -0,0 +1,24 @@
+{{/*
+Define annotations as provided in values
+*/}}
+{{- define "alfresco-common.nginx.annotations" }}
+{{- range $annotation, $value := .ingress.annotations }}
+  {{- if ne $annotation "nginx.ingress.kubernetes.io/server-snippet" }}
```

Comment:
we are also forbidding to use server-snippet here, we should probably document it here rather than in the other